### PR TITLE
Previous replaced projection spans should not expand

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@amcasey, @tmat, @kevinh-ms, @ManishJayaswal

File: src/InteractiveWindow/Editor/InteractiveWindow.cs [hunk 1]
```diff
@@ -1917,7 +1917,7 @@ private void ReplaceProjectionSpans(ReadOnlyCollection<SnapshotSpan> oldProjecti
 
         private static ITrackingSpan CreateTrackingSpan(SnapshotSpan snapshotSpan)
         {
-            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Positive);
+            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Negative);
```
Comment (amcasey) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Do I understand correctly that this is the only behavioral change?

Comment (amcasey) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Was this the only place where the two tracking modes didn't agree?  If so, can they be combined?

Comment (cston) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Yes, this is the only change in behavior. We use { Negative, Positive } elsewhere, when creating a tracking span for the leading edge of the input buffer.

**Comment by amcasey** [comment]
:+1:

**Pull request merged**